internal/dockercompose/client.go

```diff
@@ -162,6 +163,42 @@ func (c *cmdDCClient) Down(ctx context.Context, p model.DockerComposeProject, st
 	return nil
 }
 
+func (c *cmdDCClient) Rm(ctx context.Context, specs []model.DockerComposeUpSpec, stdout, stderr io.Writer) error {
+	if len(specs) == 0 {
+		return nil
+	}
+
+	// To be safe, we try not to run two docker-compose downs in parallel,
+	// because we know docker-compose up is not thread-safe.
+	c.mu.Lock()
+	defer c.mu.Unlock()
+
+	p := specs[0].Project
+	args := c.projectArgs(p)
+	if logger.Get(ctx).Level().ShouldDisplay(logger.VerboseLvl) {
+		args = append(args, "--verbose")
+	}
+
+	var serviceNames []string
+	for _, s := range specs {
+		serviceNames = append(serviceNames, s.Service)
+	}
+
+	args = append(args, []string{"rm", "--stop", "--force"}...)
```

Comment:
Feels like they need `"--please", "--but-really"`

In non-trolling comments, `rm` doesn't make any comment about timeout, while `stop` + `down` both have a timeout argument that's 10 seconds by default. I'm GUESSING (though too lazy to look at the source right now) that means when `rm --stop` is called, you'll get the default 10 second timeout.

I don't think there's much we can do around that, I'm only noting because if you disable a service that's super slow to respond to `SIGTERM` and then disable another shortly after, you might not see any activity on the second for ~10 secs, but 🤷

Reply:
Oh, good note. I don't want to distract by digging into this now, but I've added this comment to the code:
```
	// `docker-compose rm` does not support a `--timeout` option, so it possibly defaults to 10,
	// like `docker-compose stop` or `docker-compose down`.
	// If it turns out this command's timeout is too long, we might want to change this to first
	// call `docker-compose stop --timeout $NUM`, to do the presumably slow part under a smaller
	// timeout.
```